static/src/stylesheets/module/content/_content.global.scss

```diff
@@ -61,13 +61,20 @@
         margin: 0;
         position: relative;
         padding-left: 18px;
-        @include mq(leftCol) {
-            padding-left: 15px;
-        }
-        .i {
+
+        .i, .inline-icon {
             position: absolute;
             left: 0;
         }
+
+        @include mq(leftCol) {
+            padding-left: 15px;
```

Comment:
Nope you are right. Just that I didn't write it, only moved it :smile: but I will refactor now.